cognite/neat/rules/importer/_base.py

```diff
@@ -1,103 +1,50 @@
-import warnings
 from abc import ABC, abstractmethod
-from pathlib import Path
-from typing import Literal, overload
 
 import pandas as pd
 from pydantic_core import ErrorDetails
 
-from cognite.neat.rules import exceptions
-from cognite.neat.rules.models import TransformationRules
-from cognite.neat.rules.parser import RawTables, from_tables
-from cognite.neat.utils.utils import generate_exception_report
+from cognite.neat.rules.models.raw_rules import RawRules
+from cognite.neat.rules.models.rules import Rules
 
 
 class BaseImporter(ABC):
-    def __init__(self, spreadsheet_path: Path | None = None, report_path: Path | None = None):
-        self.spreadsheet_path = spreadsheet_path
-        if self.spreadsheet_path and not report_path:
-            self.report_path = self.spreadsheet_path.parent / "report.txt"
-        else:
-            self.report_path = Path.cwd() / "report.txt"
+    """
+    BaseImporter class which all importers inherit from.
+    """
 
     @abstractmethod
-    def to_tables(self) -> RawTables:
-        raise NotImplementedError
-
-    def to_spreadsheet(
-        self,
-        filepath: Path | None = None,
-        validate_results: bool = True,
-    ) -> None:
-        filepath = filepath or self.spreadsheet_path
-        if not filepath:
-            raise ValueError("No filepath given")
-        tables = self.to_tables()
-
-        with pd.ExcelWriter(filepath) as writer:
-            tables.Metadata.to_excel(writer, sheet_name="Metadata", header=False, index=False)
-
-            # Add helper row to classes' sheet
-            pd.DataFrame(
-                data=[("Data Model Definition", "", "", "", "State", "", "", "Knowledge acquisition log", "", "", "")]
-            ).to_excel(writer, sheet_name="Classes", index=False, header=False, startrow=0)
-            tables.Classes.to_excel(writer, sheet_name="Classes", index=False, header=True, startrow=1)
-
-            # Add helper row to properties' sheet
-            pd.DataFrame(
-                data=[
-                    ["Data Model Definition"]
-                    + [""] * 5
-                    + ["Start"]
-                    + [""] * 3
-                    + ["Knowledge acquisition log"]
-                    + [""] * 3
-                ]
-            ).to_excel(writer, sheet_name="Properties", index=False, header=False, startrow=0)
-            tables.Properties.to_excel(writer, sheet_name="Properties", index=False, header=True, startrow=1)
-
-            if not tables.Prefixes.empty:
-                tables.Prefixes.to_excel(writer, sheet_name="Prefixes", index=False)
-
-        if validate_results and self.report_path:
-            self._validate_rules(tables)
-
-    @overload
-    def to_rules(self, return_report: Literal[False] = False) -> TransformationRules:
+    def __init__(self):
```

Comment:
If the init method is not used lets just delete it. Currently you are requiring all importers to implement an init method. No need to do that in case you have an config free importer.